pkg/mapper/dynamicfile/dynamicfile.go

```diff
@@ -165,6 +186,24 @@ func (ms *DynamicFileMapStore) CallBackForFileLoad(dynamicContent []byte) error
 		return err
 	}
 	ms.saveMap(userMappings, roleMappings, awsAccounts)
+
+	// emit load latency and if first time, skip emit and set to false to avoid instance bounce issue
```

Comment:
could we describe more about what does `instance bounce` mean ?

Reply:
addressed, updated the comment